packages/template-retail-react-app/app/components/recommended-products/index.jsx

```diff
@@ -73,7 +73,7 @@ const RecommendedProducts = ({zone, recommender, products, title, shouldFetch, .
             getRecommendations(recommender, _products)
             return
         }
-    }, [zone, recommender, _products])
+    }, [zone, recommender])
```

Comment:
@unandyala I found a weird behavior in cart page when the recommendation section "You May also like" refetch every time we change quantity in the basket, which I find unnecessary. This is also the reason the screen reader read out lout so much updates than we expect. Hence I decided to remove the _product to not refetch on quantity change. These products will be fetched once when cart page is loaded.

By removing this, we don't need to have `isLiveRegion` for the price components. Let me know if you have any concern. Thanks